saleor/dashboard/product/forms.py

```diff
@@ -323,14 +343,22 @@ def clean_seo_description(self):
     def save(self, commit=True):
         attributes = self.get_saved_attributes()
         self.instance.attributes = attributes
+
+        price = self.cleaned_data["price"]
+        if not price.currency:
```

Comment:
How it could happen that price has `None` as currency?

Reply:
Not sure. It happened a few times in the dashboard. I will try to dig around to find a consistent way of reproducing it.

Reply:
Right, his might be an issue somewhere else that is propagating to forms.